assets/queries/ansible/aws/cloudtrail_logging_disabled/query.rego

```diff
@@ -0,0 +1,25 @@
+package Cx
+
+CxPolicy[result] {
+    document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+    task["community.aws.cloudtrail"].enable_logging == false
```

Comment:
Ansible also allows "false" and "no" we should check for these issues using 
isAnsibleFalse(answer) {
     lower(answer) == "no"
} else {
     lower(answer) == "false"
} else {
     answer == false
}